be/src/exprs/agg/java_window_function.h

```diff
@@ -27,7 +28,7 @@ class ConvertDirectBufferVistor : public ColumnVisitorAdapter<ConvertDirectBuffe
     Status do_visit(const BinaryColumn& column);
 
     template <typename T>
-    Status do_visit(const vectorized::FixedLengthColumnBase<T>& column) {
+    Status do_visit(const vectorized::FixedLengthColumn<T>& column) {
```

Comment:
does DecimalV3Column needs the do_visit method?

Reply:
UDF does not support decimal types at this time